<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
    </property>
    <property name="text">
     <string>BAŞARILI:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 47, 14);</string>
    </property>
    <property name="text">
     <string>KAÇIRILAN :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1361</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/new/prefix1/images/back.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>KAÇIRILAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>BAŞARILI</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>641</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/new/prefix3/images/2f26afa2-4eb5-4a84-b88a-c79f51a4d717-or.jpg);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="easy">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="medium">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>165</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="hard">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>210</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="imposible">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>290</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix3/images/playbuton.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="geriSayimLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>30 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ARKA PLAN RESMİNİ AŞAĞIDAKİ TOOLBARDAN DEĞİŞTİREBİLİRSİNİZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix3/images/saat.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="action1_arkaplan"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionarkaplan2"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="arkaplan3"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <action name="action1_arkaplan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix3/images/depositphotos_67982979-stock-illustration-horizontally-seamless-game-background.jpg</normaloff>:/new/prefix3/images/depositphotos_67982979-stock-illustration-horizontally-seamless-game-background.jpg</iconset>
   </property>
   <property name="text">
    <string>1.arkaplan</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionarkaplan2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix3/images/fruit-ninja-6.jpg</normaloff>:/new/prefix3/images/fruit-ninja-6.jpg</iconset>
   </property>
   <property name="text">
    <string>arkaplan2</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="arkaplan3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/images/back.jpg</normaloff>:/new/prefix1/images/back.jpg</iconset>
   </property>
   <property name="text">
    <string>arka plan 3</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
